wave_viewer.m

```diff
@@ -1041,6 +1041,16 @@ function  update_pitch_ax(pitch_ax,wave_ax,ampl_ax,p)
 gram_ax = axes(p.guidata.taxesPanel);
 axinfo = new_axinfo('gram',params{1}.taxis,params{1}.faxis,axdat,gram_ax,hzbounds4plot,wave_axinfo.name,params{1}.taxis(1),params{1}.taxis(end),params,wave_axinfo.p);
 set(gram_ax,'UserData',axinfo);
+
+% hide formants
+if ~p.plot_params.show_formants
```

Comment:
`if isfield(p.plot_params,'yes_show_formants') && ~p.plot_params.yes_show_formants`

Reply:
Added this. Checking for `isfield` wasn't actually necessary in my first commit, but it is now.